docs/cars.py

```diff
@@ -0,0 +1,211 @@
+  exceptions = {}
+  for car_exception in CAR_EXCEPTIONS:
+    if CP.carFingerprint in car_exception.cars:
+      exceptions[car_exception.column] = car_exception
+  return exceptions
+
+
+CARS_MD_OUT = os.path.join(BASEDIR, "docs", "CARS_generated.md")
+
+# TODO: which other makes?
+MAKES_GOOD_STEERING_TORQUE = ["toyota", "hyundai", "volkswagen"]
+CAR_EXCEPTIONS = [
+  CarException([TOYOTA.LEXUS_CTH, TOYOTA.LEXUS_ESH, TOYOTA.LEXUS_NX, TOYOTA.LEXUS_NXH, TOYOTA.LEXUS_RX,
+                TOYOTA.LEXUS_RXH, TOYOTA.AVALON, TOYOTA.AVALONH_2019, TOYOTA.COROLLA, TOYOTA.HIGHLANDER,
+                TOYOTA.HIGHLANDERH, TOYOTA.PRIUS, TOYOTA.PRIUS_V, TOYOTA.RAV4, TOYOTA.RAV4H, TOYOTA.SIENNA],
+               "When disconnecting the Driver Support Unit (DSU), openpilot Adaptive Cruise Control (ACC) will replace "
+               "stock Adaptive Cruise Control (ACC). NOTE: disconnecting the DSU disables Automatic Emergency Braking (AEB).",
+               Column.LONGITUDINAL, star="half"),
+  CarException([TOYOTA.CAMRY, TOYOTA.CAMRY_TSS2, TOYOTA.CAMRYH],
+               "28mph for Camry 4CYL L, 4CYL LE and 4CYL SE which don't have Full-Speed Range Dynamic Radar Cruise Control.",
+               Column.FSR_LONGITUDINAL),
+  CarException([GM.ESCALADE_ESV, GM.VOLT, GM.ACADIA],
+               "Requires an [OBD-II](https://comma.ai/shop/products/comma-car-harness) car harness and [community built ASCM harness]"
+               "(https://github.com/commaai/openpilot/wiki/GM#hardware). NOTE: disconnecting the ASCM disables Automatic Emergency Braking (AEB).",
+               Column.MODEL),
+  CarException([VOLKSWAGEN.SKODA_KAMIQ_MK1],
+               "Not including the China market Kamiq, which is based on the (currently) unsupported PQ34 platform.",
+               Column.MODEL),
+  CarException([VOLKSWAGEN.PASSAT_MK8],
+               "Not including the USA/China market Passat, which is based on the (currently) unsupported PQ35/NMS platform.",
+               Column.MODEL),
+  CarException([VOLKSWAGEN.ARTEON_MK1, VOLKSWAGEN.ATLAS_MK1, VOLKSWAGEN.TRANSPORTER_T61, VOLKSWAGEN.TCROSS_MK1,
+                VOLKSWAGEN.TROC_MK1, VOLKSWAGEN.TAOS_MK1, VOLKSWAGEN.TIGUAN_MK2],
+               'Model-years 2021 and beyond may have a new camera harness design, which isn\'t yet available from the comma '
+               'store. Before ordering, remove the Lane Assist camera cover and check to see if the connector is black '
+               '(older design) or light brown (newer design). For the newer design, in the interim, choose "VW J533 Development" '
+               'from the vehicle drop-down for a harness that integrates at the CAN gateway inside the dashboard.',
+               Column.MODEL),
+  CarException([TOYOTA.PRIUS, TOYOTA.PRIUS_V],
+               "An inaccurate steering wheel angle sensor makes precise control difficult.",
+               Column.STEERING_TORQUE, star="half"),
+]
+
+CAR_VIDEOS = {
+  HYUNDAI.ELANTRA_2021: "https://youtu.be/_EdYQtV52-c",
```

Comment:
In the description for this video it says "hyundai elantra hybrid 2021" so I'm guessing it would be for HEV